### PR TITLE
PSMDB-1306 Release notes 4.4.23-22

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	new file:   docs/release_notes/4.4.23-22.md
	modified:   docs/release_notes/index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release_notes/4.4.23-22.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+# Percona Server for MongoDB 4.4.23-22 (2023-08-08)
+
+| **Release date** | August 8, 2023 |
```
Comment (sandraromanchenko) [inline docs/release_notes/4.4.23-22.md]:
```suggestion
| **Release date** | August 14, 2023 |
```
Release was moved, pls fix

File: docs/release_notes/4.4.23-22.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Find the full list of changes in the [MongoDB 4.4.23 Community Edition release notes](https://www.mongodb.com/docs/manual/release-notes/4.4/#4.4.23---jul-13--2023).
+
+## New Features
+
+* [PSMDB-1291](https://jira.percona.com/browse/PSMDB-1291) - Add the ability to specify the AWS Security Token Service (STS) endpoint for authentication
+
+## Bugs Fixed
+
+* [PSMDB-1280](https://jira.percona.com/browse/PSMDB-1280) - Improve PSMDB behavior on client disconnect when the `$backupCursorExtend` is opened
+
```
Comment (sandraromanchenko) [inline docs/release_notes/4.4.23-22.md:33]: PSMDB-1289 is also included, pls add it to RN

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: mkdocs-base.yml [hunk 3]
```diff
@@ -116,7 +116,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaServerforMongoDB-4.4.pdf'
           cover_title: 'Server for MongoDB Documentation 4.4'
-          cover_subtitle: 4.4.22-21 (June 6, 2023)
+          cover_subtitle: 4.4.23-22 (August 8, 2023)
```
Comment (sandraromanchenko) [inline mkdocs-base.yml]:
```suggestion
          cover_subtitle: 4.4.23-22 (August 14, 2023)
```

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

**Review by janwieremjewicz — APPROVED** [review]

**Review by igorsol — APPROVED** [review]
LGTM

**Pull request merged**